src/transformers/models/auto/modeling_auto.py

```diff
@@ -152,6 +154,7 @@
     [
         # Model with LM heads mapping
         ("nystromformer", "NystromformerForMaskedLM"),
+        ("imagegpt", "ImageGPTForCausalLM"),
```

Comment:
This is unrelated to this new model addition and should be done in its own PR.

Reply:
This is a mistake, I've removed it.